coa-helm/templates/cbas-api-deploy.yaml

```diff
@@ -0,0 +1,4 @@
+{{- include "cbas-libchart.cbas-api-deploy" (list . "coa-helm.cbas-api-deploy") -}}
+{{- define "coa-helm.cbas-api-deploy" -}}
+# Overrides go here
+{{- end -}}
```

Comment:
I tried this and didn't find a way with a small-medium amount of effort, but I will do another research pass and see if I can make it work. I agree that it would be much better the way you suggest.

Reply:
Success!